behave/features/steps/common_steps.py

```diff
@@ -34,9 +34,15 @@ def do_test(*args):
 def click_on_hyperlink(context, hyperlink_text):
```

Comment:
Why does this need to return the link?  Would it become stale after clicking it?

Reply:
I am using this step elsewhere as a check to see if it went to the right place. It isn;t being returned in the test I am running. Would appreciate views on whether this is correct way to do this. https://github.com/ministryofjustice/cla-end-to-end-tests/blob/0222691b1a9eb386c068ebfa2e5c44e25cf1b2ae/behave/features/steps/alternative_help.py#L54

Reply:
I suppose I would actually change the assert, we are just checking that we have been taken to the new url specifically FALA.  So we could just assert that we are on FALA.